Solutions/GoogleCloudPlatformIAM/Hunting Queries/GCPIAMChangedRoles.yaml

```diff
@@ -0,0 +1,32 @@
+id: 0ab42aac-2c86-443c-80fa-ef8cdd0d997e
+name: GCP IAM - Changed roles
+description: |
+  'Query searches for roles' odifications.'

```

Comment:
I think there should be modifications. in place of odifications.. Please correct.

Reply:
fixed